Library/Formula/cowpatty.rb

```diff
@@ -12,6 +12,8 @@ class Cowpatty < Formula
   end
 
   def install
+    # May fail on parallel builds with a race condition, see #45150
```

Comment:
Maybe. I emailed the author a bug report using contact info on the website. Last update to cowpatty was 6 years ago, though, so who knows? Will let you know if they answer. Would be an easy `Makefile` fix, if they're still taking patches.